__tests__/__fixtures__/multiple-module-no-beforeeach-import-exists-output.js

```diff
@@ -12,7 +12,7 @@ module('Acceptance | browse acceptance test', function (hooks) {
   hooks.beforeEach(function () {
     let testMetadata = getTestMetadata(this);
     testMetadata.filePath =
-      'test/__fixtures__/multiple-module-no-beforeeach-import-exists-code.js';
+      '__tests__/__fixtures__/multiple-module-no-beforeeach-import-exists-code.js';
```

Comment:
It'd probably be nicer to have a `getFixture` util that took the filename of the fixture, and returned it. This way you wouldn't have all this repetitive fixture pathing throughout the tests.

Reply:
I'll create a new issue for this.